web/packages/teleterm/src/ui/Search/pickers/ActionPicker.tsx

```diff
@@ -125,12 +126,11 @@ export function ActionPicker(props: { input: ReactElement }) {
     }
   }
 
-  const NoResultsComponent = (
-    <NoResults
-      clusters={clustersService.getRootClusters()}
-      inputValue={inputValue}
-    />
-  );
+  const ExtraComponent = inputValue
+    ? attempts.every(a => isEmptySearch(a) && hasFinished(a)) && (
```

Comment:
I wish `useAsync` was more strict, `data` should be defined only for the `success` variant so that it's not possible to inspect `data` without inspecting the status first, which for me is the biggest strength of using such data structure in the first place.